src/solver/conv_hip_implicit_gemm_bwd_v4r1_xdlops.cpp

```diff
@@ -30,10 +30,6 @@
 #include <miopen/solver/implicitgemm_util.hpp>
 #include <cstddef>
 
-/// Disable ConvHipImplicitGemmBwdDataV4R1Xdlops by default.
```

Comment:
This W/A must be not removed but narrowed to FP32 in this PR. Otherwise the next tuning round will bring the problem back.

And I am not sure that ConvHipImplicitGemmBwdDataV4R1Xdlops is good for FP16.

`test_regression_half_mi100` has failed because the failing test case is intended for ConvHipImplicitGemmBwdDataV4R1Xdlops specifically. This test should have been disabled in #1207 (and in #1208) under the `WORKAROUND_ISSUE_1206`